examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -23,16 +22,18 @@ document.title = containerId;
 const dataObjectId = "dice";
 
 async function start(): Promise<void> {
-    const service = new TinyliciousService();
+    const containerConfig: FluidCreateContainerConfig = {
+        id: containerId,
+        dataObjects: [KeyValueDataObject],
+        initialDataObjects: [[dataObjectId, KeyValueDataObject]],
```

Comment:
I'm not a huge fan of this syntax and am open to ideas. I like this more than object in array and I generally don't like using Maps for these things. This is at lease readable.

Reply:
a new user seeing this will wonder why you need to specify KeyValueDataObjects twice...why not just infer KVDO is required since its part of initialDataObjects

Reply:
object notation? This ensures that there is only 1 value per key

```js
        initialDataObjects: {
          [dataObjectId]: KeyValueDataObject,
          [dataObjectId2]: KeyValueDataObject,
          [someOtherId]: SomeOtherDataObject
        },
```

Reply:
Gets rid of some of the errors below, which I like

Reply:
> I generally don't like using Maps for these things.

Is there a particular reason?

Reply:
I really like @micahgodbolt proposal. I was thinking `{id:string, dataObject:IFluidStaticDataObjectClass}[]` which I don't like because it makes things huge.

```
[
  { 
    id: "foo1"
    dataObject: Foo
  },
  {  
    id: "bar1"
    dataObject: Bar
  }
]
```

Reply:
I didn't address your comment Tyler. I read it first as why I don't like object syntax. Maps are okay but they usually just add unnecessary clutter. Ex. The syntax would looks something like this which makes me ask "why map?"

```
const containerConfig: FluidCreateContainerConfig = {
        id: containerId,
        dataObjects: [KeyValueDataObject],
        initialDataObjects: new Map([dataObjectId, KeyValueDataObject]),
}
```